rpm/dtc.spec

```diff
@@ -1,45 +1,107 @@
+%define     __dtc_version     1.4.6
+
 Name:       dtc
-Summary:    Linux Kernel device tree compiler
-Version:    0.0.0
-Provides:   dtc
+Version:    %{__dtc_version}
 Release:    1
+Summary:    Device Tree Compiler
+License:    GPLv2+
 Group:      Development/Tools
-URL:        https://git.kernel.org/pub/scm/utils/dtc/dtc.git
+URL:        https://devicetree.org/
+
 Source0:    %{name}-%{version}.tar.bz2
-License:    GPLv2
 
-BuildRequires: flex
-BuildRequires: bison
+BuildRequires:     gcc
+BuildRequires:     make
+BuildRequires:     flex
+BuildRequires:     bison
+BuildRequires:     swig
+
+%if 0%{?dtc_build_python_bindings:1}
```

Comment:
Lets not add python package if we don't need to, less stuff to maintain.

Reply:
Dropped `python2-libfdt`.